### PR TITLE
refactor(entire project): add refactor/css on top on development

### LINKED ISSUE #241: Code formatting
I noticed that some files use 4 spaces and some use 2 spaces. We also use " in some cases and ' in others. What do you all think about using ESLint so that we have some consistency in how we format the code? I also found this, which I haven't tried yet, but it looks pretty nice.

https://www.npmjs.com/package/standard

The nice thing about using a linter is that it makes the code cleaner and easier to read.

### LINKED ISSUE #247: Add ESLint
For consistency, we should add linting rules for the code base.

### LINKED ISSUE #348: Refactor Entire CSS
Right now CSS is split into Emotion Styled Components, Module CSS, and regular CSS.

Change to SCSS

Add SCSS folder structure

### PR DESCRIPTION
**Pull request by davidholyko**
## Description

Fixes #348 
Fixes #247 
Fixes #241 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation change (updates to README and related files)

### CODE REVIEW

File: src/components/Common/subcomponents/OrganizationCardSaveButton.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const OrganizationCardSaveButton = props => {
+  const { saveExist, saveItem } = props;
+  const buttonSign = saveExist ? String.fromCharCode(0x2713) : '+';
+
```
Comment (RajChaudhry) [inline src/components/Common/subcomponents/OrganizationCardSaveButton.js]: Add animation through the onClick here. Via:

`        let classes = [
            styles['cbutton--effect-radomir__after'],
            styles['cbutton--effect-radomir__cbutton--click__after'],
            styles['cbutton__after'],
        ];

        this.setState({
            animateButtonInside: styles['animate-button-click'],
            animateButtonOutside: classes,
        });
        setTimeout(() => {
            this.setState({
                animateButtonInside: '',
                animateButtonOutside: [''],
            });
        },
            500
        );`

File: src/css/component-styles/AppContainer.scss [hunk 2]
```diff
@@ -19,16 +19,18 @@
...
 
-.feedbackContainer {
-  z-index: 999;
-  color: #000;
-  position: fixed;
+.feedback-container {
+  background-color: #e6e6e6;
   bottom: 0px;
+  display: flex;
+  flex-direction: column;
```
Comment (RajChaudhry) [inline src/css/component-styles/AppContainer.scss]: Changing feedback-container to flex messed up the positioning of the message. I attached the picture of the extra space highlighted in red.
edit

File: src/css/component-styles/OrganizationCard.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  font-size: $font-size-lg;
+  font-weight: bold;
+  letter-spacing: 2px;
+  width: calc(100% - 25px);
+}
+
+.organization-card-body {
+  display: flex;
+  flex-direction: column;
+  padding: $spacing-md;
```
Comment (RajChaudhry) [inline src/css/component-styles/OrganizationCard.scss]: missing `line-height: 1.5;` from the original OrganizationCardLayout.js file, May be the reason why the address part is a bit offset.

Comment (RajChaudhry) [reply src/css/component-styles/OrganizationCard.scss]: unless `flex` covers that, i am not 100% sure

File: src/components/Common/subcomponents/OrganizationCardSaveButton.js (hunk 1 above)
Comment (davidholyko) [reply src/components/Common/subcomponents/OrganizationCardSaveButton.js]: i was planning on replacing the animation in a future PR. right now theres a lot of logic to produce that effect and the future PR would handle it in a cleaner way

File: src/css/component-styles/OrganizationCard.scss (hunk 3 above)
Comment (davidholyko) [reply src/css/component-styles/OrganizationCard.scss]: fixed

File: src/css/component-styles/AppContainer.scss (hunk 2 above)
Comment (davidholyko) [reply src/css/component-styles/AppContainer.scss]: fixed

**Comment by davidholyko** [comment]
small changes UI:

* I removed the badge from the save resource button and had a color change on the button to give UI feedback

* Removed button animation. It can be re-implemented in a cleaner way in a future PR.

Core functionality is all still there and other miscellaneous css tasks can be good first PR's

File: docs/issue_template.md [hunk 4]
```diff
@@ -1,41 +0,0 @@
-<!--
```
Comment (timtheguy) [inline docs/issue_template.md]: Why are we removing this?

File: src/components/Common/subcomponents/OrganizationCardSaveButton.js (hunk 1 above)
Comment (timtheguy) [reply src/components/Common/subcomponents/OrganizationCardSaveButton.js]: If you want to update the animation in a future PR, just make sure to create a corresponding issue referencing this conversation so there's no confusion as to why it happened this way!

File: docs/issue_template.md (hunk 4 above)
Comment (davidholyko) [reply docs/issue_template.md]: good catch, this is one of the things i missed when doing the merge conflicts

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+  "workbench.colorCustomizations": {
```
[the remark this answers is not in the document]
Comment (davidholyko) [reply .vscode/settings.json]: its left off now

File: src/components/Common/subcomponents/OrganizationCardSaveButton.js (hunk 1 above)
Comment (davidholyko) [reply src/components/Common/subcomponents/OrganizationCardSaveButton.js]: i added a todo item in the comments on the savebutton. will open up the issue once the PR is merged in

**Pull request merged**